components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/RecoveryConfigImpl.java

```diff
@@ -64,7 +64,7 @@ public int getOrder() {
 
         Map<String, String> nameMapping = new HashMap<>();
         nameMapping.put(IdentityRecoveryConstants.ConnectorConfig.NOTIFICATION_BASED_PW_RECOVERY,
-                "Notification based password recovery");
+                "Enable password recovery with email");
```

Comment:
We can change the notification mechanism channel, can't we? (email/Sms) in that case, will this label make sense in the product-is.